src/zope/schema/tests/test__bootstrapfields.py

```diff
@@ -958,11 +961,35 @@ def test_fromUnicode_miss(self):
         self.assertRaisesWrongType(txt.fromUnicode, txt._type, deadbeef)
 
     def test_fromUnicode_hit(self):
-
         deadbeef = u'DEADBEEF'
         txt = self._makeOne()
         self.assertEqual(txt.fromUnicode(deadbeef), deadbeef)
 
+    def test_normalization(self):
+        deadbeef = unicodedata.normalize('NFD', u'ÄÖÜ')
```

Comment:
Is it possible to write that that using escape sequences, or decoding a byte string or the like? Then you don't have to rely on `#coding` cookies, and everyone having an editor that's properly set up to handle UTF-8, plus having fonts that can display those characters in the editor, and browser, and so on.

Since normalization is involved, and there are different ways to configure that also, I worry that this problem could be made worse as different people save the file.

Reply:
I'm very thankful for that idea! Changed it.